packages/launch/src/launch/index.tsx

```diff
@@ -21,26 +21,11 @@ const FocusedLaunchModal: React.FunctionComponent< Props > = ( { onClose } ) =>
 			overlayClassName="launch__focused-modal-overlay"
 			bodyOpenClassName="has-focused-launch-modal"
 			onRequestClose={ onClose }
-			title=""
+			title={ __( 'Complete setup' ) }
+			icon={ <Icon icon={ wordpress } size={ 36 } /> }
```

Comment:
This line causes a TypeScript error which interrupts `yarn install`, and therefore I can't test the rest of the changes

Reply:
You're right 😢

Reply:
Can you do the following change in your node_modules folder please?

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49156/files

Reply:
Oh, I see. Applying these changes fixes the issue.

I also think this may be the main reason why most of the CI tasks fail